.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Also include relevant motivation & further context of what the PR is all about. List any dependencies that are required for this change.
+
+Fixes #(Insert Issue Here Without Brackets)
+
+## Changes Include
+
+Please delete options that are not relevant.
```

Comment:
```suggestion
<!-- Please delete options that are not relevant  -->
```
We need this as a comment, to inform the PR author. otherwise, it will pollute the PR description 😅